src/pages/getting-started/helpdesk-apps/index.mdx

```diff
@@ -0,0 +1,158 @@
+- Extend HelpDesk user interfaces
+- Create your own custom actions for agents in HelpDesk
+- Automate tasks with webhooks
+- Let users sign in to your service via [Accounts SDK](/authorization/sign-in-with-livechat#accounts-sdk)
+
+If you plan to make your app public for everyone, check out our <a href="https://www.livechat.com/marketplace/" target="_blank">distribution tool</a>.
+
+# Creating HelpDesk apps
+
+## Step-by-step guide
+
+If you want to create a simple HelpDesk app in the Developer Console, follow the instructions we’ve prepared. We'll use an app named Your Buddy as our example.
+
+### Step 1 Getting started
+
+To start, you need to check if you are signed in to the <Placeholder id="DEVELOPER_CONSOLE_URL" />. You can <a href="https://accounts.livechat.com/signup" target="_blank">sign up here</a>. However, your LiveChat or HelpDesk credentials should log you in automatically. 
+
+The first step in the Developer Console is to go to the **Apps** tab and click the **Create new app +** button. Give your app a name and click the **Continue** button. The application's name can be changed later.
+
+<img
+  alt="Creating HelpDesk apps"
+  src="/images/getting-started/helpdesk-apps/livechat-helpdesk-apps-creating new app.png"
+  width="1459px"
+  height="771px"
+/>
+
+Next, you need to decide which product you want to build your app for. In this tutorial guide, we will cover HelpDesk. Want to build for LiveChat instead? See the [tutorial](/getting-started/livechat-apps).
+
+<div style={{maxWidth: '450px'}}>
+  <img src="/images/getting-started/helpdesk-apps/livechat-helpdesk-apps-choosing-product.png" alt="Creating HelpDesk apps choosing the product" width="557px" height="386px"/>
+</div>
+
+If you do not have a HelpDesk account, you will be asked to activate it. If you want to activate HelpDesk and start a free trial, click **Activate HelpDesk** button.
+
+<div style={{maxWidth: '450px'}}>
+  <img src="/images/getting-started/helpdesk-apps/livechat-helpdesk-apps-activating account.png" alt="Creating HelpDesk apps activating account" width="279px" height="230px"/>
+</div>
+
+💡 **Need a helping hand?** Right after signing in to the Developer Console, you can launch the **App Wizard**. We go there with guidelines for the LiveChat app creation process, but it can also be very helpful while exploring HelpDesk.
+
+### Step 2 Adding HelpDesk Widgets block
+
+You can form your app using components called Building Blocks. Each Building Block works differently. If you want to get more information about Building Blocks, click the **Read more** redirect or go to [App guides](/getting-started/app-guides).
+
+In this guide, we will use the HelpDesk Widgets building block. Widgets are web apps that can interact and communicate with the HelpDesk App. They are loaded inside the HelpDesk App interface.
+
+<img
+  alt="Creating HelpDesk apps building blocks"
+  src="/images/getting-started/helpdesk-apps/livechat-helpdesk-apps-building-blocks.png"
```

Comment:
to be changed as the name of the Onboarding & Settings has been changed and In-app Upgrades has been removed.